haas/cli.py

```diff
@@ -233,19 +245,19 @@ def do_request(method, url, data={}):
         `url` - The url to make the request to
         `data` - the body of the request.
     """
-    return check_status_code(http_client.request(method, url, data=data))
+    check_status_code(http_client.request(method, url, data=data))
 
 def do_put(url, data={}):
-    return do_request('PUT', url, data=json.dumps(data))
+    do_request('PUT', url, data=json.dumps(data))
 
 def do_post(url, data={}):
-    return do_request('POST', url, data=json.dumps(data))
+    do_request('POST', url, data=json.dumps(data))
 
 def do_get(url):
-    return do_request('GET', url)
+    do_request('GET', url)
 
 def do_delete(url):
-    return do_request('DELETE', url)
+    do_request('DELETE', url)
```

Comment:
That was on purpose - those functions never actually returned anything,
and nobody was checking anyway.